database/migrations/2021_02_11_161257_alter_invoices_table_add_source.php

```
@@ -24,37 +22,63 @@ public function up()
+                title VARCHAR NOT NULL,
+                comment TEXT,
+                price INTEGER NOT NULL,
+                offer_id INTEGER UNSIGNED,
+                invoice_id INTEGER UNSIGNED,
+                type VARCHAR,
+                quantity INTEGER,
+                product_id VARCHAR,
+                created_at TIMESTAMP,
+                updated_at TIMESTAMP,
+                deleted_at TIMESTAMP
+            )');
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\Illuminate\Support\Facades\DB' in method 'up'. (undefined)

(StaticAccess)